src/test/util/mod.rs

```diff
@@ -118,16 +118,24 @@ impl TestClient {
     pub async fn create_user(
         &self,
         user: &str,
-        pwd: &str,
+        pwd: impl Into<Option<&str>>,
```

Comment:
Passwords aren't used for X.509 authentication, so I made this parameter optional. Additionally, I wanted to use a separate database for the X.509 tests to ensure that the test wasn't accidentally passing by using the SCRAM credentials, so I added a new parameter to specify a database (which still defaults to "admin").